pmd-cataloger/src/main/java/sfdc/sfdx/scanner/pmd/Main.java

```diff
@@ -35,26 +35,32 @@ public Main(Dependencies dependencies) {
   public static void main(String[] args) {
 
     final Main main = new Main(new Dependencies());
-    main.mainInternal(args);
+    final int exitCode = main.mainInternal(args)? 0 : 1;
+    System.exit(exitCode);
```

Comment:
@jfeingold35 Added exitCode based on our conversation. Thanks for the suggestion :)